week2/week_2_day_4_all_textbook_exercises.Rmd

```diff
@@ -0,0 +1,111 @@
+---
+title: "week_2_day_3_textbook_exercises"
+output: html_document
+date: '2022-06-09'
+---
+
+###############Chapter 2 (ISRS)
+
+####2.2 
+
+(a)
+```{r}
+control_patients_died <- 30/34
+control_patients_died
+
+treated_patients_died <-45/69
+treated_patients_died
+
+```
+
+(b)
+          (i)
+          The claims being tested are:
+          Null hypothesis: Survival rate is not effected by treatment. Observations of higher survival rate                            occured due to chance. 
+          Alternate hypothesis: Survival rate is effected by treatment. Observations of higher survival                                    rate didn't occur due to chance. 

```

Comment:
Idk what happened to your formatting here, but in the future you can use markdown syntax for bulleted lists to avoid this weirdness:
```markdown
- bullet one
  - sub bullet
    - sub sub bullet
```
etc